isofit/radiative_transfer/radiative_transfer.py

```diff
@@ -252,35 +240,43 @@ def get_L_atm(self, x_RT: np.array, geom: Geometry) -> np.array:
                 r = RT.get(x_RT, geom)
                 rdn = r["rhoatm"]
                 if RT.rt_mode == "transm":
+                    # transform atmospheric path reflectance to path radiance
                     rdn = (self.solar_irr * self.coszen) / np.pi * rdn
                 L_atms.append(rdn)
         return np.hstack(L_atms)
 
-    def get_L_down_transmitted(self, x_RT: np.array, geom: Geometry) -> np.array:
-        """Get the interpolated total downward atmospheric transmittance.
-        Thermal_downwelling already includes the transmission factor. Also
-        assume there is no multiple scattering for TIR.
+    def get_E_down(self, x_RT: np.array, geom: Geometry) -> np.array:
+        """Get the interpolated direct and diffuse downward fluxes on the sun-to-surface path.
+        Thermal_downwelling already includes the transmission factor.
+        Also assume there is no multiple scattering for TIR.
 
         Args:
             x_RT: radiative-transfer portion of the statevector
             geom: local geometry conditions for lookup
 
         Returns:
-            interpolated total downward atmospheric transmittance
+            interpolated direct and diffuse downward fluxes on the sun-to-surface path
         """
-        L_downs = []
+        E_downs = []
         for RT in self.rt_engines:
             if RT.treat_as_emissive:
                 r = RT.get(x_RT, geom)
                 rdn = r["thermal_downwelling"]
-                L_downs.append(rdn)
+                E_downs.append(rdn)
             else:
                 r = RT.get(x_RT, geom)
-                rdn = r["transm_down_dir"] + r["transm_down_dif"]
+                E_down_dir = r["transm_down_dir"]
+                E_down_dif = r["transm_down_dif"]
                 if RT.rt_mode == "transm":
-                    rdn = (self.solar_irr * self.coszen) / np.pi * rdn
-                L_downs.append(rdn)
-        return np.hstack(L_downs)
+                    # transform downward transmittance to downward flux
+                    E_down_dir = (
+                        (self.solar_irr * self.coszen) / np.pi * r["transm_down_dir"]
```

Comment:
@pgbrodrick @davidraythompson I'm scaling the direct flux by the TOA solar zenith angle here to conform to the rescaling I mentioned above in line 190.

Reply:
Resolved as per conversation with the team in a previous tag up.